.angular-cli.json

```diff
@@ -21,6 +21,7 @@
       "prefix": "app",
       "styles": [
         "assets/bootstrap.min.css",
+        "../node_modules/font-awesome/css/font-awesome.css",
```

Comment:
doesnt this need to be in assets?

Reply:
Yes, copy to assets and use a minimized lib